app/src/main/java/com/github/se/icebreakrr/model/tags/Tags.kt

```diff
@@ -0,0 +1,21 @@
+package com.github.se.icebreakrr.model.tags
+
+data class TagsCategory(
+    val name: String = "",
+    val color: String = "#FFFFFFFF",
+    val subtags: List<String> = emptyList()
```

Comment:
Question: shouldn't at least `name` be mandatory?

Reply:
This is just a constructor used in some cases where I want to instantiate a `TagsCategory` so that I can assign to it a value later, so I don't use the name at this moment